Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.m

```diff
@@ -61,10 +76,22 @@ - (instancetype)initWithBridge:(nonnull RCTBridge *)bridge
     _animationNodes = [NSMutableDictionary new];
     _eventDrivers = [NSMutableDictionary new];
     _activeAnimations = [NSMutableSet new];
+    _shadowViewProps = [[NSArray alloc] init];
+    _operationsInBatch = [NSMutableArray new];
```

Comment:
`_uiManagerOperationQueue` to go with `enqueueUpdateViewOnUIManager`

Reply:
Done